src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,5 +1,47 @@
 import React from 'react';
 
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
-);
+import { Post } from '../../types/Post';
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+
+import './PostInfo.scss';
+
+type Props = {
+  post: Post;
+};
+
+export const PostInfo: React.FC<Props> = ({ post }) => {
+  const {
+    id,
+    title,
+    body,
+    user,
+    comments,
+  } = post;
+
+  return (
+    <div className="PostInfo" key={id}>
```

Comment:
```suggestion
    <div className="PostInfo" >
```
There is no need to add key here